OMCompiler/Compiler/FrontEnd/DAEUtil.mo

```diff
@@ -909,9 +909,13 @@ algorithm
     local
       DAE.Exp r;
     case (SOME(DAE.VAR_ATTR_REAL(start = SOME(r)))) then r;
+    case (SOME(DAE.VAR_ATTR_REAL(start = NONE()))) then DAE.RCONST(0.0);
     case (SOME(DAE.VAR_ATTR_INT(start = SOME(r)))) then r;
+    case (SOME(DAE.VAR_ATTR_INT(start = NONE()))) then DAE.ICONST(0);
     case (SOME(DAE.VAR_ATTR_BOOL(start = SOME(r)))) then r;
+    case (SOME(DAE.VAR_ATTR_BOOL(start = NONE()))) then DAE.BCONST(false);
     case (SOME(DAE.VAR_ATTR_STRING(start = SOME(r)))) then r;
+    case (SOME(DAE.VAR_ATTR_STRING(start = NONE()))) then DAE.SCONST("");
     case (SOME(DAE.VAR_ATTR_ENUMERATION(start = SOME(r)))) then r;
```

Comment:
What is the default value for enumerations?

Reply:
The first enumeration value according to [the specification](https://specification.modelica.org/master/class-predefined-types-and-declarations.html#predefined-types-and-classes):
```Modelica
parameter EnumType start = e1;       // Initial value
```